ajax/libs/rsuite/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "rsuite",
+  "filename": "rsuite.min.js",
+  "version": "3.0.1",
+  "description": "A suite of react components",
+  "homepage": "http://rsuitejs.com/",
```

Comment:
Lets use https

Reply:
We only need to host the version after 3.0.0,So how should I configure it? AndI checked the 3.0.0 and later versions. Their directories are the same as my current configuration.Thank you very much.

Reply:
You pattern already fetches only version 3.0.0 and above. Lets wait for @sufuf3 @PeterDaveHello @extend1994 answer to my question above

Reply:
@sashberd  OK,Thank you very much.
@sufuf3 @PeterDaveHello @extend1994 Can you help me?

Reply:
@hiyangguo Yes, it's alright not to add the versions the library author or main  contributor(s) think unnecessary.
By the way, could you tell if the file `dist/suite.js` of 0.0.16 and the below versions is compatible with a browser? I think the answer is yes, but it's better to check with the one who is more familiar with the library.
Once this is confirmed and url is updated to https, the PR can be merged! :slightly_smiling_face:

Reply:
@extend1994  I am this library main contributor , I'm sure that before 3.0.0 version of this lib is not required for hosting. And I will update later, thank you very much.